tests/cases/fourslash/fullCaseCompletions1.ts

```diff
@@ -0,0 +1,87 @@
+/// <reference path="fourslash.ts" />
+
+// Basic tests
+
+// @newline: LF
+//// enum E {
+////     A = 0,
+////     B = "B",
+////     C = "C",
+//// }
+//// // Mixed union
+//// declare const u: E.A | E.B | 1;
+//// switch (u) {
+////     case/*1*/
+//// }
+//// // Union enum
+//// declare const e: E;
+//// switch (e) {
+////     case/*2*/
+//// }
+//// enum F {
+////     D = 1 << 0,
+////     E = 1 << 1,
+////     F = 1 << 2,
+//// }
+////
+//// declare const f: F;
+//// switch (f) {
+////     case/*3*/
+//// }
+
+verify.completions(
+    {
+        marker: "1",
+        isNewIdentifierLocation: false,
+        includes: [
+            {
+                name: "case E.A: ...",
+                source: completion.CompletionSource.SwitchCases,
+                sortText: completion.SortText.GlobalsOrKeywords,
+                insertText:
+`case E.A:
+case E.B:
+case 1:`,

```

Comment:
I feel like I would expect snippet tab stops after each `:`; what do you think?

Reply:
I haven't considered that before. I guess I'll add this and try it out and ask others to try it to see if it's annoying or not.